extensions/emojicloud.py

```diff
@@ -80,13 +80,15 @@ async def main(ctx: lightbulb.Context) -> None:
     )
     for p in listofimages:
         if p[3][0] == "<":  # Custom Emoji have "<" in the beginning
-            thumbnail = Image.open(
-                os.path.join(
-                    script_dir, "..", emoji_cache.get_file_name_if_cached(p[3])
+            file_name = await emoji_cache.get_file_name(p[3], ctx.bot)
```

Comment:
Add a comment [3] index is the emoji - (size, x-coord, y-coord, emoji)

Really should have a class for the that tbh but add it to the tech debt.

Reply:
This is what happens when you get the intern to write code I guess